modules/ipcscript.js

```diff
@@ -60,8 +123,6 @@ IPCScript.prototype.info = function() {
   return {
     'uuid': this.uuid,
     'version': this.addonVersion,
```

Comment:
Yes, but:
https://github.com/greasemonkey/greasemonkey/pull/2259/files#diff-e07419236596eaa414a5954c7c90c385R151

However, this does not help:
`return new IPCScript(script, gGreasemonkeyVersion);`

GM_info.version = 'unknown';